terraform/mssql/provision/main.tf

```diff
@@ -55,11 +55,31 @@ resource "aws_db_instance" "db_instance" {
   storage_type           = var.storage_type
   iops                   = contains(local.valid_storage_types_for_iops, var.storage_type) ? var.iops : null
 
+  parameter_group_name = aws_db_parameter_group.db_parameter_group.name
+
   lifecycle {
     prevent_destroy = true
   }
 
   timeouts {
     create = "60m"
   }
+}
+
+resource "aws_db_parameter_group" "db_parameter_group" {
```

Comment:
Hey! I discovered this pipeline error:
https://hush-house.pivotal.io/teams/cloud-service-broker/pipelines/aws-csb/jobs/run_terraform_tests/builds/754#L647c1266:428

Was caused by this PR. 
This PR added a new resource but didn't modify the corresponding Terraform tests:

https://github.com/cloudfoundry/csb-brokerpak-aws/blob/607ccda0896ee4b74547be9cc3aa98a983699437/terraform-tests/mssql_test.go#L65-L72

I'll be uploading a fix